src/commands/ip_allow.ts

```diff
@@ -0,0 +1,203 @@
+import yargs from 'yargs';
+import { CommonProps, ProjectScopeProps } from '../types';
+import { writer } from '../writer.js';
+import { Project, ProjectUpdateRequest } from '@neondatabase/api-client';
+import { projectUpdateRequest } from '../parameters.gen.js';
+import { log } from '../log.js';
+
+const IP_ALLOW_FIELDS = [
+  'id',
+  'name',
+  'IP_addresses',
+  'primary_branch_only',
+] as const;
+
+export const command = 'ip-allow';
+export const describe = 'Manage IP Allow';
+export const builder = (argv: yargs.Argv) => {
+  return argv
+    .usage('$0 ip-allow <sub-command> [options]')
+    .options({
+      'project-id': {
+        describe: 'Project ID',
+        type: 'string',
+        demandOption: true,
```

Comment:
Don't make it demanded option, this way context would not work.

Reply:
Good catch. Removed.